articles/introduction-to-reinforcement-learning/index.md

```diff
@@ -0,0 +1,109 @@
+### Table of Contents
+
+1. [Introduction](#introduction)
+2. [Key Concepts](#key-concepts)
+3. [Most Used Reinforcement Learning Algorithms](#most-used-reinforcement-learning-algorithms)
+4. [Applications of Reinforcement Learning](#applications-of-reinforcement-learning)
+5. [Final Thoughts](#final-thoughts)
+6. [References](#references)
+
```

Comment:
You could add a section that details the pre-requisites that the reader would need to read before they dive into this article.

Reply:
Fixed